### PR TITLE
Display the speaker name in each story

### PR DESCRIPTION
**Pull request by AliTopal89**

### CODE REVIEW

File: rails/app/javascript/components/StoryList.jsx [hunk 1]
```diff
@@ -63,7 +63,8 @@ class StoryList extends Component {
             style={style}
           >
             <div className="speakers">
-              <img src={story.speaker.picture_url} alt={story.speaker.name} title={story.speaker.name}/>
+              <img src={story.speaker.picture_url}/>
```
Comment (mirandawang) [inline rails/app/javascript/components/StoryList.jsx]: We might want to keep the alt and title properties here for accessibility reasons. What do you think?

Comment (AliTopal89) [reply rails/app/javascript/components/StoryList.jsx]: I checked it out with it and without it. I really didn't see the difference but I have no problem with adding that back in.

Comment (mirandawang) [reply rails/app/javascript/components/StoryList.jsx]: I think when you use screen readers, that’s how it knows how to describe the images. I haven’t tested it out myself though.

Comment (AliTopal89) [reply rails/app/javascript/components/StoryList.jsx]: oh ok I understand what you are saying. Adding that back in now.

**Comment by mirandawang** [comment]
Thanks Ali looks great!!

**Pull request merged**